src/did/did-ion-resolver.ts

```diff
@@ -1,17 +1,22 @@
 import crossFetch from 'cross-fetch';
 import { DidMethodResolver, DidResolutionResult } from './did-resolver.js';
 
+// supports fetch in: node, browsers, and browser extensions.
+// uses native fetch if available in environment or falls back to a ponyfill.
+// 'cross-fetch' is a ponyfill that uses `XMLHTTPRequest` under the hood.
+// `XMLHTTPRequest` cannot be used in browser extension background service workers.
+// browser extensions get even more strict with `fetch` in that it cannot be referenced
+// indirectly.
+const fetch = globalThis.fetch ?? crossFetch;
+
 /**
  * Resolver for ION DIDs.
  */
 export class DidIonResolver implements DidMethodResolver {
-  // cross-platform fetch
-  private fetch = crossFetch;
-
   /**
    * @param resolutionEndpoint optional custom URL to send DID resolution request to
    */
-  constructor (private resolutionEndpoint: string = 'https://discover.did.microsoft.com/1.0/identifiers/') { }
+  constructor (private resolutionEndpoint: string = 'https://beta.discover.did.microsoft.com/1.0/identifiers/') { }
```

Comment:
Thanks for the update @mistermoe, sanity: should we still use the GA endpoint here though? Ot it stopped working?

Reply:
It stopped working 😢. Check out the screenshot i posted in the PR description of curl failing

Reply:
Hm... just tried and it works again. I guess it's still best to reply on their GA endpoint instead of beta? Though you seemed to find the beta endpoint more reliable....

Reply:
all good @thehenrytsai! thanks for figuring it out. went ahead and updated the resolution endpoint